integr8ly/keycloak-client
# MGDAPI-6014 Resolve CVEs in KC Client repo

# Issue Link

Jira: https://issues.redhat.com/browse/MGDAPI-6014

# What

Resolve CVEs reported by Dependabot by updating dependencies.

# Verification Steps

`checkout branch`

Run `make build`

Run ` make unit/tests`

Ensure that there are no errors
